doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,184 @@
+
+Save the configuration and restart the packet forwarder:
+
+```bash
+$ systemctl restart nas_lgw
+```
+
+If your configuration was successful, your gateway will connect to {{% tts %}} after a couple of seconds. You can easily check if your gateway has connected using the Console.
+
+The rest of the document discusses further topics that are not required for the gateway to work, but may be useful for your deployments.
+
+
+## Troubleshooting
+
+If the gateway does not connect to {{% tts %}} after a few minutes, issue a `reboot` command, or disconnect and reconnect the power supply to power-cycle the gateway.
+
+You can access the gateway system logs using journalctl. Use the `-f` flag to follow the journal (i.e. do not exit and display new logs as they arrive), `-u nas_lgw` to only show the logs related to the packet forwarder, and `-n 1000` to show the last 1000 log lines.
+
+```bash
+$ journalctl [-f] [-u nas_lgw] [-n 1000]
+```
+
+
+## Extras: VPN
+
+Your gateway will most likely be installed on the field and connect to the network server via the Internet (using cellular data or an Ethernet connection). Most likely this means that your gateway will be sitting behind a NAT, so connecting to it directly (for example, trying to SSH should something break) is impossible.
+
+Thankfully, your gateway comes with a pre-installed OpenVPN client. In order to connect the gateway to your VPN, first generate a new client configuration file for your server and name it `CLIENT.conf`. Setting up the VPN server itself and generating the client configuration file is out-of-scope for this document.
+
+Copy the file to the gateway with `scp`:
+
+```bash
+$ scp CLIENT.conf root@<GatewayIP>:CLIENT.conf
+```
+
+And then, enable the openvpn client for your configuration:
+
+```bash
+$ ssh root@<GatewayIP>
+> systemctl start openvpn@CLIENT.service
+> systemctl status openvpn@CLIENT.service    # <--- verify that it works
+> systemctl enable openvpn@CLIENT.service    # <--- enable so that the service will restart on reboot
+```
+
+Your gateway should now be connected to your VPN server.
+
+> NOTE: The installed OpenVPN client version may be old and not support the `tls-crypt` configuration command. You need to use `tls-auth` instead. You have to configure your VPN server accordingly.
+
+
+## Extras: Logging
+
+The gateway logs rotate when they reach about 15M in size, which means that you will generally not be able to access very old logs. At times of dense traffic (e.g. ~1000s of devices) this typically means that you only have logs available for the previous 2 hours. If you want to keep historical data (for whatever reason), then you can forward the logs to an other server using `netcat`:
+
+```bash
+$ journalctl -f | nc server-hostname port-number &
```

Comment:
This does not work in a general case, i.e. if I connect over SSH, run this command and logout, it will exit.
Either provide a generic solution, or (preferrable IMO) just remove it and let the users figure it out themselves - keep the information about rotation though. Probably it can go to troubleshooting section

Reply:
Edited that bit and moved it to the Troubleshooting section above. It should be a bit better now